addons/mail/static/src/models/message/message.js

```diff
@@ -267,6 +287,33 @@ function factory(dependencies) {
             return `${this.modelName}_${data.id}`;
         }
 
+        /**
+         * @returns {boolean}
+         */
+        _computeCanBeDeleted() {
+            const isMailChannelMessage = this.originThread && this.originThread.model === 'mail.channel'
+            const hasRights = session.is_admin || this.isCurrentPartnerAuthor;
+            if (isMailChannelMessage) {
+                return hasRights && this.message_type === 'comment';
+            } else {
+                return hasRights && this.is_note;
+            }
+        }
+
+        /**
+         * @returns {boolean}
+         */
+        _computeCanStarBeToggled() {
+            if (this.isTemporary) {
+                return false;
+            }
+            const isMailChannelMessage = this.originThread && this.originThread.model === 'mail.channel'
+            if (isMailChannelMessage && this.message_type !== 'notification') {
```

Comment:
I just transcribed the conditions that were previously in the template, as far as why it's currently the case, I don't know. Since there was no change on this front in the spec I assumed that the current behaviour should be preserved.

Reply:
It's not really important either way, I guess it's fine to keep it then. I had no idea it was there, it just feels an arbitrary limitation.